spec/models/queries/work_packages/columns/work_package_column.rb

```diff
@@ -0,0 +1,39 @@
+#-- copyright
```

Comment:
As the file has no `_spec.rb` ending, it will not be executed by running `rspec [some folder]`